file.py

```diff
@@ -1 +1,2 @@
 print("hello jonny")
```

Comment:
Please capitalize Jonny (he forgot to do that)

Reply:
Done